en/08_Changelogs/6.0.0.md

```diff
@@ -479,6 +486,9 @@ The [`BuildTask`](api:SilverStripe\Dev\BuildTask) class is now a subclass of `Po
 
 As a result of this, any `BuildTask` implementations in your project or module will need to be updated. The upgrade will likely look like this in most cases:
 
+<details>
+<summary>Click to see the diff</summary>
```

Comment:
Decided to make this easier to skip for people who are just scrolling down the page. Same with another longer diff later on.